frontend/www/js/omegaup/components/arena/Runs.vue

```diff
@@ -308,25 +311,35 @@
                     class="btn-link dropdown-item"
                     @click="onRunDetails(run)"
                   >
-                    {{ T.wordsDetails }}
+                    {{ T.arenaRunsActionsDetails }}
                   </button>
                   <button
                     v-if="showRejudge"
-                    data-actions-rejudge
+                    :data-actions-rejudge="run.guid"
                     class="btn-link dropdown-item"
                     @click="$emit('rejudge', run)"
                   >
-                    {{ T.wordsRejudge }}
+                    {{ T.arenaRunsActionsRejudge }}
                   </button>
                   <div class="dropdown-divider"></div>
-                  <button
-                    v-if="showDisqualify"
-                    data-actions-disqualify
-                    class="btn-link dropdown-item"
-                    @click="$emit('disqualify', run)"
-                  >
-                    {{ T.wordsDisqualify }}
-                  </button>
+                  <template v-if="showDisqualify">
+                    <button
+                      v-if="run.type !== 'disqualified'"
```

Comment:
se puede mostrar esto sólo si

```suggestion
                      v-if="run.type === 'normal'"
```

para que la condición sea idéntica a la que tiene el backend?

Reply:
Listo.